.github/scripts/local-build.sh

```diff
@@ -7,36 +7,151 @@
 
```

Comment:
I tried testing this script locally seeing as you awesomely updated the contrib docs, but got this error.  I'm not sure what might be wrong but seems like the command arg parser is not reading `--git-repo-path .` for some reason..

```shell
~/dev/repos/voulage$ .github/scripts/local-build.sh --git-repo-path . --extension .github/scripts/ext-debian.sh --package-name regolith-i3status-rust --package-url https://github.com/regolith-linux/regolith-i3status-rust.git --package-ref debian_source --distro ubuntu --codename jammy --stage unstable
mkdir: cannot create directory ‘/pkgbuild’: Permission denied

```

Reply:
Nice catch! https://github.com/regolith-linux/voulage/pull/38/commits/6b8341984ab0de828a14c71d655d013955c49a17 should fix it.

Reply:
Worked perfectly